src/styles/styles.js

```diff
@@ -3077,13 +3077,26 @@ const styles = (theme) => ({
     },
 
     cardMenuItem: {
-        paddingLeft: 8,
+        paddingLeft: 0,
```

Comment:
@pac-guerreiro any idea why we chnaged this to 0? it is causing design issues in other files

Reply:
Discussed 1:1, this should have stayed 8px, fixed